### PR TITLE
Copy-edit README.md Issue: #11168

### PR DESCRIPTION
**Pull request by LeaSak**
Hello. I've copy-edited the README.md. These are just suggestions. Besides correcting some grammar and punctuation, I tried to reduce the length of some sentences and the use of passive to help improve readability. I also fixed the link to the API docs.

Pull request for issue: #11168
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:
   - Count of stars:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
Great!

**Comment by maruilian11** [comment]
Could you modify the issue number in commit msg to #11168? And I think everything is wonderful. :smile:

**Comment by LeaSak** [comment]
Hi @maruilian11. I managed to modify the commit message and learn a bit more about git in the meantime :smiley:

File: README.md [hunk 1]
```diff
@@ -26,35 +26,35 @@
...
+CDNJS is a free open-source project to organize and provide all famous front-end web development resources to developers via a fast CDN infrastructure without usage limitations and fees. We want to help individual library/framework developers distribute their projects, and web developers supercharge their websites! With our great free CDN service, developers can focus on their projects and website development. Developers no longer have to spend time worrying about how to set-up a CDN for projects or website assets. We hope to make web development easier, as well as your websites and the WWW faster and safer.
```
Comment (PeterDaveHello) [inline README.md]: hmmm, I suppose we usually use the common term `free and open source project` instead of `free open-source project`?

Comment (LeaSak) [reply README.md]: Hi @PeterDaveHello. Sure, I can fix that if you like. Also, in my PR I capitalized the acronym url to URL in all cases in the README body except for in the Table of Contents. The comments mentioned not to touch this section manually. Should I keep this change or switch it back to how it was beforehand to be consistent with the Table of Contents item?

Comment (PeterDaveHello) [reply README.md]: Thanks, let's change it back!

It's fine if you'd like to update the Table of Contents :)

Comment (LeaSak) [reply README.md]: Ok, great. Just one last question from a beginner. Should I squash the new commit with the above changes with my first commit on my branch after I make my changes?

Comment (petrgazarov) [inline README.md]: > and provide all famous front-end web development resources to developers

~all famous~ -> popular

Actors and celebrities can be described as "famous", but with software libraries it sounds a bit comical

Comment (LeaSak) [reply README.md]: Agree. I'll swap the word. No problem.

Comment (maruilian11) [reply README.md]: @LeaSak Yes, we hope the commits can be as clean as possible. Could you please squash them? Thanks.

Comment (LeaSak) [reply README.md]: Thanks @maruilian11 for helping me. I just squashed the commits.

File: README.md [hunk 2]
```diff
@@ -26,35 +26,35 @@
...
 
 ## Contributing
 
-To contribute to CDNJS, please refer to the [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md), it should contain most of the things you'll need to get your contribution started!
+To contribute to CDNJS, please refer to [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md). It should contain most of the things you'll need to get your contribution started!
 
-**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
+**Working on your first Pull Request?** Learn how from this *free* series: [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
```
Comment (PeterDaveHello) [inline README.md]: This was copied from http://makeapullrequest.com/ actually, but I won't mind if we have better wording.

**Comment by LeaSak** [comment]
Thanks for everyone's help and patience. This is exciting.

**Comment by PeterDaveHello** [comment]
@LeaSak Thank you for your contribution 😄

**Pull request merged**